functions/display-logic.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * Displays a single story on the front page.
+ **/
+function display_front_page_story( $story, $css_class='', $show_vertical=false, $thumbnail_size='frontpage-story-thumbnail', $heading='h3' ) {
+	if ( !$story ) { return false; }
+
+	$thumbnail = null;
+	$thumbnail_id = get_front_page_story_thumbnail_id( $story );
+
+	if ( $thumbnail_id ) {
+		$thumbnail = wp_get_attachment_image(
+			$thumbnail_id,
+			$thumbnail_size,
+			false,
+			array(
+				'class' => 'fp-feature-img center-block img-fluid',
+				'alt' => '' // Intentionally blank to avoid redundant story title announcement
+			)
+		);
+	}
+
+	$title = wptexturize( $story->post_title );
+
+	$description = '';
+	if ( $story_description = get_post_meta( $story->ID, 'story_description', true ) ) {
+		$description = wptexturize( strip_tags( $story_description, '<b><em><i><u><strong>' ) );
+	}
+	elseif ( $story_subtitle = get_post_meta( $story->ID, 'story_subtitle', true ) ) {
+		$description = wptexturize( strip_tags( $story_subtitle, '<b><em><i><u><strong>' ) );
+	}
+
+	$vertical = null;
+	if ( $show_vertical ) {
+		$vertical = get_the_category( $story->ID );
+		if ( $vertical ) {
+			$vertical = $vertical[0];
+			$vertical = wptexturize( $vertical->name );
+		}
+	}
+
+	ob_start();
+?>
+<article class="fp-feature <?php echo $css_class; ?>">
+	<a class="fp-feature-link" href="<?php echo get_permalink( $story->ID ); ?>">
+		<?php if ( $thumbnail ): ?>
+		<div class="fp-feature-img-wrap">
+			<?php echo $thumbnail; ?>
+
+			<?php if ( $show_vertical && $vertical ): ?>
+			<span class="fp-vertical">
+				<?php echo $vertical; ?>
+			</span>
+			<?php endif; ?>
+		</div>
+		<?php endif; ?>
+	</a>
+	<div class="fp-feature-text-wrap">
+		<<?php echo $heading; ?> class="fp-feature-title">
+			<a class="fp-feature-link" href="<?php echo get_permalink( $story->ID ); ?>">
+				<?php echo $title; ?>
+			</a>
+		</<?php echo $heading; ?>>
+		<div class="fp-feature-description">
+			<?php echo $description; ?>
+		</div>
+	</div>
+</article>
+<?php 	return ob_get_clean();
+}
+
+
+/**
+ * Displays a single Today article on the front page.
+ **/
+function display_front_page_today_story( $article ) {
+	$url = $article->get_link();
+	$title = $article->get_title();
+	$publish_date = $article->get_date('m/d');
+
+	ob_start();
+?>
+<article class="fp-today-feed-item">
+	<a class="fp-today-item-link" href="<?php echo $url; ?>">
+		<div class="publish-date"><?php echo $publish_date; ?></div>
+		<?php echo $title; ?>
+	</a>
+</article>
+<?php 	return ob_get_clean();
+}
+
+
+/**
+ * Displays a single event item on the front page.
+ **/
+function display_front_page_event( $event ) {
+	$start = strtotime( $event['starts'] );
+	$description = substr( strip_tags( $event['description'] ), 0, 250 );
+	if ( strlen( $description ) == 250 ) {
```

Comment:
```suggestion
	if ( strlen( $description ) === 250 ) {
```

Should use a strict equality here.

Reply:
Updated in both `display-logic.php` files.